conf/officialsAndNominees.routes

```diff
@@ -200,5 +200,10 @@ POST       /nominee-payments
 GET        /nominee-payments/change                                             controllers.nominees.IsIndividualNomineePaymentsController.onPageLoad(mode: Mode = CheckMode)
 POST       /nominee-payments/change                                             controllers.nominees.IsIndividualNomineePaymentsController.onSubmit(mode: Mode = CheckMode)
 
+GET        /nominee-changed-address                                             controllers.nominees.IsIndividualNomineePreviousAddressController.onPageLoad(mode: Mode = NormalMode)
+POST       /nominee-changed-address                                             controllers.nominees.IsIndividualNomineePreviousAddressController.onSubmit(mode: Mode = NormalMode)
+GET        /nominee-changed-address/change                                      controllers.nominees.IsIndividualNomineePreviousAddressController.onPageLoad(mode: Mode = CheckMode)
+POST       /nominee-changed-address/change                                      controllers.nominees.IsIndividualNomineePreviousAddressController.onSubmit(mode: Mode = CheckMode)
+
```

Comment:
please move this (as well as nominee-payments) to above the nominee-organisation-name route to keep it tidy